recipe/recipe.yaml

```
@@ -0,0 +1,133 @@
+context:
+  version: "1.7.5"
+
+recipe:
+  name: cvxpy-split
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/cvxpy/cvxpy/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: 01b589b51a49ac8d1e4a41bf98ba4ed8048ce1ade2a8344f9b4fdb5574f0e9f4
+    patches:
+      # test that fails due to timing issues on slow CI agents when in emulation
+      - patches/0001-loosen-time-limit-in-test_scipy_mi_time_limit_reache.patch
+
+build:
+  number: 1
+
+outputs:
+  - package:
+      name: cvxpy-base
+      version: ${{ version }}
+    build:
+      script:
+        # if not in release mode, this may lead to issues, see
+        # https://github.com/cvxpy/cvxpy/issues/1690
+        - if: unix
+          then:
+            - export CXXFLAGS="${CXXFLAGS} -DNDEBUG"
+            - rm $SRC_DIR/pyproject.toml
+          else:
+            - del %SRC_DIR%\pyproject.toml
+        - python -m pip install . --no-deps -vvv
+    requirements:
+      build:
+        - if: build_platform != target_platform
+          then:
+            - python
+            - cross-python_${{ target_platform }}
+            - numpy
+        - ${{ compiler("c") }}
+        - ${{ stdlib("c") }}
+        - ${{ compiler("cxx") }}
+      host:
+        - python
+        - pybind11
+        - numpy
+        - pip
+        - setuptools
+      run:
+        - python
+        - scipy >=1.1.0
+    tests:
+      - python:
+          imports:
+            # public interface is defined (see #75) to be the content of
+            # https://github.com/cvxpy/cvxpy/blob/master/cvxpy/__init__.py
+            - cvxpy
+
+  - package:
+      name: cvxpy
+      version: ${{ version }}
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - ${{ pin_subpackage('cvxpy-base', exact=True) }}
+        - osqp >=0.6.2
+        - clarabel >=0.5
+        - cffi
+        - scs >=3.2.4
+    tests:
+      - python:
+          imports:
+            - cvxpy
+            - cvxpy.cvxcore
+            - cvxpy.cvxcore.python
+      - requirements:
+          run:
+            - pip
+            - pytest
+            # issues with newer scipy and other libraries
+            # - cvxopt
+            - hypothesis
+            - scipy
+        files:
+          source:
+            - cvxpy/tests
+        script:
+          interpreter: python
+          content:
+            - 'import sys'
+            - 'import subprocess'
+            - 'import platform'
+            - 'tests_to_skip = "_not_a_real_test"'
+            - '# accuracy failure on PPC, likely due to emulation'
+            - 'if platform.machine().lower() == "ppc64le":'
+            - '    tests_to_skip += " or (TestDqcp and test_basic_multiply)"'
```

Use proper v1 selectors
```suggestion
            - if: ppc64le
              then: 
                - 'tests_to_skip += " or (TestDqcp and test_basic_multiply)"'
```